lib/couchdb_to_sql/changes.rb

```diff
@@ -137,9 +137,7 @@ def perform_request
       uri = URI.parse(url)
 
       # Authenticate?
-      if uri.user.present? && uri.password.present?
-        @http.set_auth(source.root, uri.user, uri.password)
-      end
+      @http.set_auth(source.root, uri.user, uri.password) if uri.user.present? && uri.password.present?
```

Comment:
I'm fine with the change, but I'm also fine with the original way this was written. Rubocop is wrong in this regard. 😛